packs/travisci/actions/setup_build.yaml

```diff
@@ -0,0 +1,14 @@
+name: setup_build
```

Comment:
It would probably be better and more explicit to have two actions - `restart_build` and `cancel_build`.